nvflare/app_common/abstract/model_persistor.py

```diff
@@ -52,6 +90,7 @@ def save_model(self, model: ModelLearnable, fl_ctx: FLContext):
         """
         pass
 
+    @abstractmethod
```

Comment:
./nvflare/app_common/np/np_model_persistor.py
./nvflare/app_opt/sklearn/joblib_model_param_persistor.py
./nvflare/app_opt/xgboost/tree_based/model_persistor.py

These class inherits from this ModelPersistor class but they did not implement the "get_model_inventory" method.

Do we want to ask all users to implement this "get_model_inventory" method?
If so, we need to change all these files as a start^

Reply:
Previously, only load_model() and save_model() need to be implemented. I changed it to keep that behavior.